ui/js/controllers/perf/alerts.js

```diff
@@ -491,8 +491,9 @@ perf.controller('AlertsCtrl', [
                 if (_.isUndefined($scope.alertSummaries)) {
                     $scope.alertSummaries = alertSummaries;
                 } else {
-                    $scope.alertSummaries = _.union($scope.alertSummaries,
```

Comment:
This is throwing an error when I go to the perfherder Alerts tab when pointed to stage (``yarn start:stage``).  Can you test this out and see why it's erroring?  Perhaps one of those objects are not in the form we expect, or null?

Reply:
Sorry, I must have missed that one. I'll take a look.

Reply:
I assume it's the same case as #3657? I don't see any error in the Alert tab.